routers/web/web.go

```diff
@@ -492,7 +493,6 @@ func registerRoutes(m *web.Route) {
 	}, reqSignIn, ctxDataSet("PageIsUserSettings", true, "AllThemes", setting.UI.Themes, "EnablePackages", setting.Packages.Enabled))
 
 	m.Group("/user", func() {
-		// r.Get("/feeds", binding.Bind(auth.FeedsForm{}), user.Feeds)
```

Comment:
Justify the commenting-out or remove :)

Reply:
Now there is `user.rss` try.gitea.io/wxiaoguang.rss

Reply:
I mean in code, not here :)

Reply:
Hmm, I didn't get the point (not native reader 😭 )  what should be done?

Reply:
Commented-out code is bad. Either remove it completely or add a comment above it to explain why it may still be good to have in the future.

Reply:
Agree, so I just removed it. Then nothing need to do?

Reply:
Yeah that's fine as well.